src/modules/CreateDrops/DropSummaryModal.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Spinner,
+  UseDisclosureProps,
+} from '@chakra-ui/react';
+import { CheckIcon } from '@chakra-ui/icons';
+
+import { RoundIcon } from '@/common/components/IconBox/RoundIcon';
+
+interface DropSummaryModalProps extends UseDisclosureProps {
+  confirmationText: string;
+  isConfirmed: boolean;
+}
+
+// TODO: enhance css after merging sushan's sign in modal branch
```

Comment:
[Q] should i turn this into a reusable modal in design system? 🤔

Reply:
This [PR](https://github.com/keypom/keypom-airfoil/pull/22/files#diff-fc780f9e227b4e37b0cbba5775d6adaf0f49fa3183b8d500de127fa5aac2e47a) adds modal to the theme. if there's a new variant we can add it there. but would be great if we wait for the [PR](https://github.com/keypom/keypom-airfoil/pull/22) to get merged first.